### ISSUE TITLE
Screen flickering when you move the mouse by certain zones of the screen

### ISSUE THREAD

**Issue by pictenon** [linuxmint/mint22-beta#53]
When you drag the mouse cursor over the zones near down at the desktop, the screen starts to flicker. 

I have tested in different computers, and both live disk and installed with updated NVIDIA drivers. This issue don't happens in the same computer with Linux Mint 21.3 loaded.

I send a video made with a mobile device, because if you record screen video with the computer app, you don't see the problem.

I send also a photo when you see in red the two lines where you if you drag there the mouse, the screen flickering triggers, the rest of the screen don't triggers any flickering. 

![Linux Mint 22 flickering screen if you drag mouse](https://github.com/linuxmint/mint22-beta/assets/18500363/edf19fd5-1e75-48a8-994e-76d87d80ef38)

https://github.com/linuxmint/mint22-beta/assets/18500363/8975cfd4-39b9-48ad-8e8c-034c4719e20e

**Comment by proulxf012** [comment]
Hello. Also had this problem. It was so bad at one point that I aborted the testing of LM22 and returned back to LM21.3. I did not remarked if it was screen zone specific.
Regards
François Proulx, Longueuil, Québec, Canada

**Comment by clefebvre** [comment]
This looks like an xorg/drivers/kernel issue. Can you reproduce this with Mint 22 MATE or Xfce?

**Comment by pictenon** [comment]
Hi Clement,

I have tested also with Mate and Xfce versions and the problem persist in these versions. Watch down the videos.

https://github.com/user-attachments/assets/3ca9ae6c-3a0b-4df7-888a-ebbd82a05874
https://github.com/user-attachments/assets/1e57a27a-6b54-42e0-b3f3-f6c263e10cb7

I think that is a issue related with nVidia driver.

What I have noticed is that in the installed and updated version of Cinnamon Mint 22 beta, when you switch the nVidia GPU Display to "performance mode", the problem disappears and works perfectly. When you switch again to "energy save mode" the problem appears again. Watch the photo.

![Cinnamon_installed_Mint_22_beta](https://github.com/user-attachments/assets/3433d3dd-a868-4d7f-b30c-336c39ba04eb)

In Mint 21.3 the screen works perfectly in both modes.

Regards,
